x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp.test.tsx

```diff
@@ -5,16 +5,26 @@
  */
 
 import React from 'react';
-import { shallowWithIntl, renderWithIntl } from '@kbn/test/jest';
 import { PingTimestamp } from './ping_timestamp';
 import { mockReduxHooks } from '../../../../lib/helper/test_helpers';
+import { render } from '../../../../lib/helper/rtl_helpers';
 import { Ping } from '../../../../../common/runtime_types/ping';
-import { EuiThemeProvider } from '../../../../../../observability/public';
+import * as observabilityPublic from '../../../../../../observability/public';
 
 mockReduxHooks();
 
+jest.mock('../../../../../../observability/public', () => {
+  const originalModule = jest.requireActual('../../../../../../observability/public');
+
+  return {
+    ...originalModule,
+    useFetcher: jest.fn().mockReturnValue({ data: null, status: 'pending' }),
+  };
+});
```

Comment:
i wish we can have a simple helper function to mock all the hooks we need in a test. 

I really don't like adding this boilerplate.

`mockHook('useFectcher' , value | function)`

Reply:
I agree. I'd be willing to tackle that in another testing util issue specifically related to the mocks.